helm/hpcc/templates/dafilesrv.yaml

```diff
@@ -91,6 +91,10 @@ spec:
 {{ include "hpcc.addCertificateVolume" (dict "root" $ "name" .name "component" "dafilesrv" "certificate" .certificate "visibility" .service.visibility) | indent 6 }}
 {{- end }}
 ---
+{{- if hasKey . "hpa" -}}
+ {{ include "hpcc.addHorizontalPodAutoscaler" (dict "name" .name "kind" "Deployment" "hpa" .hpa) }}
```

Comment:
minor: white spacing in helm templates are horrible. This doesn't generate the wrong result, but it looks like it will indent by a space (which would be wrong), I think it would look more correct (and equal) if:
```
{{- if hasKey . "hpa" }}
 {{- include "hpcc.addHorizontalPodAutoscaler" (dict "name" .name "kind" "Deployment" "hpa" .hpa) }}
{{- end }}
```